packages/reports/addon/models/bard-request/request.js

```diff
@@ -459,6 +459,27 @@ export default Fragment.extend(Validations, {
     return this.removeSort(sortObj);
   },
 
+  /**
+   * Removes the sort from the sort array using the metric model and parameters
+   *
+   * @method removeSortMetricWithParam
+   * @param {DS.Model} metricModel
```

Comment:
Can we add a description to this to say `metric metadata model`? We have two metric model, one is a fragment which is part of the request, the other is an `Ember.Object`, the metric metadata model.

I see other places in this file have this too, but can we correct this line to say `{Ember.Object}` instead of `{DS.Model}`.

Reply:
Sure, still need to familiarize myself with the project a little more though :)